mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -3811,7 +3811,7 @@ fn test_tx_history_segwit() {
 #[cfg(not(target_arch = "wasm32"))]
 fn test_tx_history_tbtc_non_segwit() {
     let passphrase = "also shoot benefit prefer juice shell elder veteran woman mimic image kidney";
-    let coins = json!([tbtc_segwit_conf(),]);
+    let coins = json!([tbtc_conf(),]);
```

Comment:
It was segwit enabled. Don't we wanna keep it the same?
https://github.com/KomodoPlatform/komodo-defi-framework/blob/29c48bb3a57cde0976ce41d5f47ffed3fa110542/mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs#L4093-L4097

Reply:
It wasn't segwit enabled, `"segwit":true` was used in config before segwit implementation meaning the coin can support segwit or not. At first for segwit we used one coin where we can convert the address format between segwit and non-segwit and this parameter was one of the params that is used to know that a coin supports this.